app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -933,6 +933,12 @@ class BrowserTabViewModel(
         ctaViewModel.registerDaxBubbleCtaDismissed(cta)
     }
 
+    fun onUserClickTopCta(cta: HomeTopPanelCta) {
```

Comment:
is this a `configuration` or a `cta`? the variable names here and in the `BrowserTabFragment` are different so it would be good to keep it consistent

Reply:
Yeah we kinda use `configuration` to call `ctas` in some places 😅 . If you have a look at the method below `onUserClickCtaOkButton` that uses cta and what receives is also called `configuration` but is a cta.